src/search/components/SearchResultItem.tsx

```diff
@@ -28,8 +28,8 @@ const SearchResultItem: FunctionComponent<SearchResultItemProps> = ({
 			date={formatDate(result.dcterms_issued)}
 			// TODO: DISABLED_FEATURE
 			// tags={[{ label: 'Redactiekeuze', id: 'redactiekeuze' }, { label: 'Partner', id: 'partner' }]}
-			viewCount={412}
-			bookmarkCount={85}
+			viewCount={(result as any).views_count || 0}
```

Comment:
Mij lijkt het dat we hier wel een type kunnen voorzien. Voorlopig een TODO bijzetten?

Reply:
fixed in: feature/bookmarks-in-workspace